extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/interceptor/TransactionalInterceptorBase.java

```diff
@@ -78,6 +79,23 @@ private Transactional getTransactional(InvocationContext ic) {
         throw new RuntimeException(jtaLogger.i18NLogger.get_expected_transactional_annotation());
     }
 
+    /**
+     * <p>
```

Comment:
no biggy but for a private method, I'd favor IDE readability over JavaDoc generation and remove these HTML tags.

Reply:
done